website/project/views/contributor.py

```diff
@@ -531,13 +531,10 @@ def send_claim_email(email, unclaimed_user, node, notify=True, throttle=24 * 360
 def notify_added_contributor(node, contributor, auth=None, throttle=None, email_template='default'):
```

Comment:
notify_added_contributor method modified to work with both preprints and node models. This is needed so preprint contributor emails can still be sent.